plugins/modules/cloud/misc/proxmox.py

```diff
@@ -292,6 +311,29 @@
      - nesting=1
      - mount=cifs,nfs
 
+- name: >
+    Create a linked clone of the template container with id 100. The newly created container with be a
+    linked clone, because no storage parameter is defined.
```

Comment:
```suggestion
    linked clone, because no storage parameter is defined
```
(The other examples shouldn't do that either; please don't modify the existing ones in this PR.)